### DISCUSSION TITLE
Access blocked: This app’s request is invalid

### DISCUSSION THREAD

**Discussion by YahuiZo** [DevSkillsHQ/discusssions#17] [Help] [answered]
Hi,

I'm preparing the environment for Chicago Crime Analysis - Level 1. When running the colab script, and visiting the URL to authorize the application, I met this:

Access blocked: This app’s request is invalid
You can’t sign in because this app sent an invalid request. You can try again later or contact the developer about this issue.
If you are a developer of this app, see error details.
Error 400: redirect_uri_mismatch

In order to check the redirec URI, I tried to find the client id mentioned in the URL in the colab console, but seems it didn't appear in my project. 

What should I do?

**Comment by fenske** [comment, marked as answer]
Hi @LauraYahui. I'm Anton from DevSkills.

Sorry that you encountered this issue.

It is related to some very recent general changes that have been done to how Google Colab communicates with BigQuery. And unfortunately, it looks like we'll need to come up with a different way for candidates to submit their solutions.

In the meantime, I've updated the instructions for how to submit your solutions. Please find them in the README file of your assignment repository.

Please let me know if you have any other questions. 

I wish you the best of luck with the rest of your interview process.

**Reply by YahuiZo** [reply]
I see. Thank you so much!
